### ISSUE TITLE
Can't use Logloss:border in python wrapper

### ISSUE THREAD

**Issue by trongdat3011** [catboost/catboost#237]
In the [official documentation](https://tech.yandex.com/catboost/doc/dg/concepts/python-reference_parameters-list-docpage/#python-reference_parameters-list), it is mentioned that i can set the label value border for the Logloss metric by modify the loss_function string. However in my code:
`model = CatBoostClassifier(loss_function='Logloss:border=0.5')`

It raises the following error:

> Key 'Logloss:border=0.5' not found in enum. Valid optionsare: 'Logloss', 'CrossEntropy', 'RMSE', 'MAE', 'Quantile', 'LogLinQuantile', 'MAPE', 'Poisson', 'MultiClass', 'MultiClassOneVsAll', 'PairLogit', 'YetiRank', 'QueryRMSE', 'QueryAverage', 'R2', 'AUC', 'Accuracy', 'Precision', 'Recall', 'F1', 'TotalF1', 'MCC', 'PairAccuracy', 'Custom', 'UserPerObjErr', 'UserQuerywiseErr'.

Am i missing something and what is the proper way to do it?

Thanks in advance.

**Comment by annaveronika** [comment]
it's a bug, I'm working on the fix. Thanks for pointing it out!

**Comment by annaveronika** [comment]
The fix will come soon, for now you could use CatBoost class for training instead of CatBoostClassifier.

**Comment by annaveronika** [comment]
Fix in on github, you can build from source using instruction: https://tech.yandex.com/catboost/doc/dg/concepts/python-installation-docpage/#python-installation
Will be on pypi in next release.

**Comment by kizill** [comment]
v0.6.1.1 release is on pypi and on releases page https://github.com/catboost/catboost/releases/tag/v0.6.1.1 , please try to upgrade.

**Comment by sbarman-mi9** [comment]
The error still persists in 0.14.2. 

core.py has conflict with loss_function and scale_pos_weight parameters. Please see.


Thanks.

**Comment by annaveronika** [comment]
We are planning on removing 'Logloss:border=value', and adding border as a separate training parameter, because currently logloss border is used in all custom metrics, that are calculated during training.

**Comment by annaveronika** [comment]
This should be fixed by now.
